Code/GraphMol/FileParsers/MolFileWriter.cpp

```diff
@@ -971,6 +971,44 @@ const std::string GetV3000MolFileAtomLine(
     atom->getPropIfPresent(common_properties::_MolFileRLabel, rLabel);
     ss << " RGROUPS=(1 " << rLabel << ")";
   }
+
+  {
+    int iprop;
+    if (atom->getPropIfPresent(common_properties::molAttachOrder, iprop) &&
+        iprop) {
+      ss << " ATTCHORD=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molAttachPoint, iprop) &&
+        iprop) {
+      ss << " ATTCHPT=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molAtomSeqId, iprop) &&
+        iprop) {
+      ss << " SEQID=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molRxnExactChange, iprop) &&
+        iprop) {
+      ss << " EXACHG=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molInversionFlag, iprop) &&
+        iprop) {
+      if (iprop == 1 || iprop == 2) ss << " INVRET=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molStereoCare, iprop) &&
+        iprop) {
+      ss << " STBOX=" << iprop;
+    }
+    if (atom->getPropIfPresent(common_properties::molSubstCount, iprop) &&
+        iprop) {
+      ss << " SUBST=" << iprop;
```

Comment:
Same question as in input: should we value check the rest of these, as we do with INVRET?